macptopbm: add page (#11316)
tldr-pages/tldr: pages/common/macptopbm.md

# macptopbm

> Read a MacPaint file as input and produce a PBM image as output.
> More information: <https://netpbm.sourceforge.net/doc/macptopbm.html>.

- Convert a MacPaint file into a PGM image:

`macptopbm {{path/to/file.mac}}`

> - Skip over extra N bytes when reading the file:
> 
> `macptopbm -extraskip N`

Reviewer: ```suggestion
- Skip over a specified number of bytes when reading the file:

`macptopbm -extraskip {{N}}`
```
How about this?

Author: Perfect!!
